sdk/monitor/opentelemetry-exporter/README.md

```diff
@@ -0,0 +1,51 @@
+
+# Azure Monitor JavaScript Exporter for OpenTelemetry
```

Comment:
you're going to get assaulted by the README dochecker in CI since it enforces common sections across all READMEs.